### PR TITLE
Fix/lint staged

### PR DESCRIPTION
**Pull request by 0x4007**
1. fix lint-staged (runs eslint and prettier on committed files only.)
2. move `/bin/` to `/utils/`
3. reimport nodemon config
4. reimport global dom typings (`*.svg` and `window.ethereum`)
5. run prettier and eslint on entire codebase (passes with ~60 warnings, no errors)
6. mobile sidebar (in the ui) enhancement. click off of sidebar to close it!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,33 +6,43 @@
...
     "build:all": "yarn workspaces foreach run build",
     "test:all": "yarn workspaces foreach run test",
-    "purge:all": "yarn workspaces foreach run purge && rimraf './node_modules'",
+    "purge:all": "yarn workspaces foreach run purge && rimraf node_modules",
     "prettier:all": "yarn workspaces foreach run prettier",
     "lint:all": "yarn workspaces foreach run lint",
-    "build:ci:cloudflare": "yarn workspaces foreach run build",
-    "build:ci:netlify": "yarn workspaces foreach run build",
-    "build:ci:fleek": "yarn workspaces foreach run build"
+    "build:ci:cloudflare": "run-s build:all",
```
Comment (cuddle-dev) [inline package.json]: do we need to install run-s here? we can just use `yarn` like `yarn build:all`

**Review by cuddle-dev — APPROVED** [review]
Looks good to me

**Comment by cuddle-dev** [comment]
can we temporarily stop changing for contracts workspaces as of now? because it has a bunch of refactors.

**Comment by 0x4007** [comment]
> can we temporarily stop changing for contracts workspaces as of now? because it has a bunch of refactors.

This is my first time using yarn workspaces but I presume that you're talking about changes in the `packages/contracts` directory.

**Comment by cuddle-dev** [comment]
> > can we temporarily stop changing for contracts workspaces as of now? because it has a bunch of refactors.
> 
> This is my first time using yarn workspaces but I presume that you're talking about changes in the `packages/contracts` directory.

Right

File: package.json (hunk 1 above)
Comment (0x4007) [reply package.json]: > do we need to install run-s here? we can just use `yarn` like `yarn build:all`

Okay good idea. So we can probably just do `    "build:ci:cloudflare": "build:all",` ?

Comment (cuddle-dev) [reply package.json]: yes. we can

**Pull request merged**